backend/app/schemas/swagger.py

```
@@ -0,0 +1,44 @@
+from fastapi import status
```

Ruff (0.11.9)

`fastapi.status` imported but unused

Remove unused import: `fastapi.status`

(F401)